diff --git a/plugins/Makefile.am b/plugins/Makefile.am
index d1b9acb..0518589 100644
--- a/plugins/Makefile.am
+++ b/plugins/Makefile.am
@@ -68,6 +68,7 @@ include experimental/money_trace/Makefile.inc
 include experimental/mp4/Makefile.inc
 include experimental/multiplexer/Makefile.inc
 include experimental/remap_purge/Makefile.inc
+include experimental/slice/Makefile.inc
 include experimental/ssl_cert_loader/Makefile.inc
 include experimental/sslheaders/Makefile.inc
 include experimental/stale_while_revalidate/Makefile.inc
