diff --git a/iocore/eventsystem/I_EThread.h b/iocore/eventsystem/I_EThread.h
index e5f1c56..3d2f9c7 100644
--- a/iocore/eventsystem/I_EThread.h
+++ b/iocore/eventsystem/I_EThread.h
@@ -297,7 +297,7 @@ public:
   virtual ~EThread();
 
   Event *schedule_spawn(Continuation *cont);
-  Event *schedule(Event *e, bool fast_signal = false);
+  Event *schedule(Event *e, bool fast_signal = true);
 
   /** Block of memory to allocate thread specific data e.g. stat system arrays. */
   char thread_private[PER_THREAD_DATA];
